django/contrib/postgres/forms/ranges.py

```diff
@@ -66,12 +68,17 @@ class IntegerRangeField(BaseRangeField):
     range_type = NumericRange
 
 
-class FloatRangeField(BaseRangeField):
+class DecimalRangeField(BaseRangeField):
     default_error_messages = {'invalid': _('Enter two numbers.')}
     base_field = forms.FloatField
     range_type = NumericRange
 
 
+class FloatRangeField(DecimalRangeField):
+    warnings.warn('FloatRangeField has been renamed to DecimalRangeField and is now deprecated')
```

Comment:
This needs to be done at `__init__` as well, right now it happens at import time.